io-sim-classes/src/Control/Monad/Class/MonadSTM/Strict.hs

```diff
@@ -131,14 +131,14 @@ writeTVar StrictTVar { tvar, invariant } !a =
 modifyTVar :: MonadSTM m => StrictTVar m a -> (a -> a) -> STM m ()
 modifyTVar v f = readTVar v >>= writeTVar v . f
 
-stateTVar :: MonadSTM m => StrictTVar m a -> (a -> (a, b)) -> STM m b
+stateTVar :: MonadSTM m => StrictTVar m a -> (a -> (b, a)) -> STM m b
```

Comment:
Maybe we should use the same tyvar names as the `stm` package, `s -> (a, s)` while we're at it?

http://hackage.haskell.org/package/stm-2.5.0.1/docs/Control-Concurrent-STM-TVar.html#v:stateTVar

Reply:
I think we should.